examples/ccc-molecule/src/basic.ts

```diff
@@ -0,0 +1,54 @@
+import { ccc } from "@ckb-ccc/shell";
+import { Buffer } from "buffer";
+
+// Basic Usage
+
+// Molecule Basic Types
+// There is only one built-in primitive type in Molecule: byte.
+console.log(ccc.mol.Bytes.encode(Buffer.from("hello")).toString());
+console.log(ccc.mol.Bytes.decode(Buffer.from("68656c6c6f", "hex")).toString());
+
+// Composite Types
+// array: An array consists of an item type and an unsigned integer.
+console.log(ccc.mol.array(ccc.mol.Bytes, 2).encode(["hello", "world"]).toString());
+console.log(ccc.mol.array(ccc.mol.Bytes, 2).decode(Buffer.from("68656c6c6f776f726c64", "hex")).toString());
+
+// struct: A struct consists of a set of named and typed fields.
+console.log(ccc.mol.struct({ a: ccc.mol.Bytes, b: ccc.mol.Bytes }).encode({ a: "hello", b: "world" }).toString());
```

Comment:
Bytes type only accepts BytesLike value (hex array). For utf8 string, consider to use ccc.mol.String.